pages/contributors.tsx

```diff
@@ -7,41 +7,64 @@ import { Spinner } from "react-bootstrap"
 import Lottie from "react-lottie"
 import Card from "../components/Cards/Card"
 
-const defaultOptions = {
+type ContributorSerializer = {
+    handle_name:string
+    name:string
+    handle_url:string
+    image:string
+    github:string
+    description:string[]
+}
```

Comment:
Rename this to something else. Serializers are used in the hackalog backend from Django Rest Framework (see [here](https://www.django-rest-framework.org/api-guide/serializers/)).

Simply, `Contributor` will work fine.

Reply:
Also mention in a comment above that objects of this type are received from the GitHub API.

Reply:
Done